lib/matplotlib/axes/_base.py

```diff
@@ -567,7 +567,11 @@ def get_window_extent(self, *args, **kwargs):
         get the axes bounding box in display space; *args* and
         *kwargs* are empty
         """
-        return self.bbox
+        bbox = self.bbox
+        x_pad = self.xaxis.get_tick_padding()
+        y_pad = self.yaxis.get_tick_padding()
+        return mtransforms.Bbox([[bbox.x0 - x_pad, bbox.y0 - y_pad],
+                                 [bbox.x1 + x_pad, bbox.y1 + y_pad]])
```

Comment:
polar axes don't have ticks -- but it does appear that this needs to be updated so it's aware of when the ticks are present or not.

It doesn't seem to crash for axes3d, though as you say, it doesn't really do the right thin other than add a bit more space.